### ISSUE TITLE
Pagination component isn't responsive for breakpoints above "lg"

### ISSUE THREAD

**Issue by Galienna** [Orange-OpenSource/Orange-Boosted-Bootstrap#1351]
### Prerequisites

- [X] I have [searched](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

When using the [pagination component](https://boosted.orange.com/docs/5.2/components/pagination/#responsive-behavior), the responsive behaviour is broken when page size is above "lg" breakpoint. All the items are displayed :
![image](https://user-images.githubusercontent.com/36480568/176905172-c0aa5cfc-911a-4ebd-9d13-4b007314e688.png)

Don't know if it's a real bug or if i'm missing a conf option...
Same for versions 5.2.0-beta1 and 5.1.3, using npm. 

### Reduced test cases

https://stackblitz.com/edit/d41sw5?file=index.html
(app block needs to be widen a bit to see the issue)

### What operating system(s) are you seeing the problem on?

Linux

### What browser(s) are you seeing the problem on?

Chrome

### What version of Boosted are you using?

5.2.0, 5.1.3

**Comment by julien-deramond** [comment]
Thanks for reporting this issue @Galienna. You're not missing options. At the moment, we only set a max items number until large breakpoint. We could introduce a new rule like that:
```scss
@include media-breakpoint-up(lg) {
  @include pagination-max-items(10);
}
```

**Comment by julien-deramond** [comment]
FYI we tackled the main issue in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1579 that will be released with the v5.2.1. We worked on https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1370 but there are still some issues of edge cases so we'll try to finish it rather for the v5.2.2 (or v5.3.0 if too complicated to resolve).
